examples/md-flexible/src/ParallelVtkWriter.cpp

```diff
@@ -191,6 +297,64 @@ void ParallelVtkWriter::createPvtuFile(const int &currentIteration) {
   timestepFile.close();
 }
 
+void ParallelVtkWriter::createPvtsFile(const int &currentIteration, const RegularGridDecomposition &decomposition) {
+  std::ostringstream filename;
+  filename << _sessionFolderPath << _sessionName << "_subdivision_" << std::setfill('0')
+           << std::setw(_maximumNumberOfDigitsInIteration) << currentIteration << ".pvts";
+
+  std::ofstream timestepFile;
+  timestepFile.open(filename.str(), std::ios::out | std::ios::binary);
+
+  if (not timestepFile.is_open()) {
+    throw std::runtime_error("Simulation::writeVTKFile(): Failed to open file \"" + filename.str() + "\"");
+  }
+  const std::array<int, 3> wholeExtent = decomposition.getDecomposition();
+  const std::array<double, 3> globalBoxMin = decomposition.getGlobalBoxMin();
+  const std::array<double, 3> globalBoxMax = decomposition.getGlobalBoxMax();
+  timestepFile << "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"no\" ?>\n";
+  timestepFile << "<VTKFile byte_order=\"LittleEndian\" type=\"PStructuredGrid\" version=\"0.1\">\n";
+  timestepFile << "  <PStructuredGrid WholeExtent=\"0 " << wholeExtent[0] << " 0 " << wholeExtent[1] << " 0 "
+               << wholeExtent[2] << "\" GhostLevel=\"0\">\n";
+  timestepFile << "    <PPointData/>\n";
+  timestepFile << "    <PCellData>\n";
+  timestepFile << "      <PDataArray type=\"Int32\" Name=\"DomainId\" />\n";
+  timestepFile << "      <PDataArray type=\"Float32\" Name=\"CellSizeFactor\" />\n";
+  timestepFile << "      <PDataArray type=\"String\" Name=\"Container\" />\n";
+  timestepFile << "      <PDataArray type=\"String\" Name=\"DataLayout\" />\n";
+  timestepFile << "      <PDataArray type=\"String\" Name=\"FullConfiguration\" />\n";
+  timestepFile << "      <PDataArray type=\"String\" Name=\"LoadEstimator\" />\n";
+  timestepFile << "      <PDataArray type=\"String\" Name=\"Traversal\" />\n";
+  timestepFile << "      <PDataArray type=\"String\" Name=\"Newton3\" />\n";
```

Comment:
These string arrays break the animation feature in Paraview. When pressing the play button, Paraview will frees at the next timestep if these containers are present.
I would suggest changing them to Int32 and mapping and and define for each container a mapping from integer to value.
This will make looking up the data harder, but at least the paraview does not freeze.